apps/websockets/models/notifications.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from config import settings
+
+user_model = settings.AUTH_USER_MODEL
+
+
+class Notification(models.Model):
+    class Type(models.TextChoices):
+        FOLLOW = "FOLLOW"
+        FOLLOW_REQUEST = "FOLLOW_REQUEST"
+        ACCEPT = "ACCEPT"
```

Comment:
What do you think, maybe it would be better to make more explicit type names, I think currently there might be some confusion about follow vs follow_request?
Also Imagine if we will have multiple ACCEPT notifications: for follow accepts and for event accepts

Reply:
What do you think about:
`NEW_FOLLOWER` (for non-private users)
`NEW_FOLLOW_REQUEST` (for private users)
`ACCEPTED_FOLLOW_REQUEST` (for your follow requests to private users)